From khali@linux-fr.org Sun Sep 24 12:25:56 2006
Date: Sun, 24 Sep 2006 21:25:52 +0200
From: Jean Delvare <khali@linux-fr.org>
To: Greg KH <greg@kroah.com>
Cc: LM Sensors <lm-sensors@lm-sensors.org>
Subject: [PATCH 23/23] hwmon: Remove Yuan Mu's address
Message-Id: <20060924212552.84abbf97.khali@linux-fr.org>
Content-Disposition: inline; filename=hwmon-remove-yuan-mu-address.patch

hwmon: Remove Yuan Mu's address

Yuan Mu no longer works at Winbond.

I wish to publicly thank Yuan for his help with Winbond hardware
monitoring chips support during the past 10 months.

Signed-off-by: Jean Delvare <khali@linux-fr.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 Documentation/hwmon/w83627ehf |    2 +-
 drivers/hwmon/w83627ehf.c     |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

--- gregkh-2.6.orig/Documentation/hwmon/w83627ehf
+++ gregkh-2.6/Documentation/hwmon/w83627ehf
@@ -9,7 +9,7 @@ Supported chips:
 
 Authors:
         Jean Delvare <khali@linux-fr.org>
-        Yuan Mu <Ymu@Winbond.com.tw>
+        Yuan Mu (Winbond)
         Rudolf Marek <r.marek@sh.cvut.cz>
 
 Description
--- gregkh-2.6.orig/drivers/hwmon/w83627ehf.c
+++ gregkh-2.6/drivers/hwmon/w83627ehf.c
@@ -2,7 +2,7 @@
     w83627ehf - Driver for the hardware monitoring functionality of
                 the Winbond W83627EHF Super-I/O chip
     Copyright (C) 2005  Jean Delvare <khali@linux-fr.org>
-    Copyright (C) 2006  Yuan Mu <Ymu@Winbond.com.tw>,
+    Copyright (C) 2006  Yuan Mu (Winbond),
                         Rudolf Marek <r.marek@sh.cvut.cz>
                         David Hubbard <david.c.hubbard@gmail.com>
 
